app/models/user.rb

```diff
@@ -44,4 +44,15 @@ def delete_friend(user)
       nil
     end
   end
+
+  def verify_friendship?(user)
+    if Friendship.where(friend_one_id: id, friend_two_id: user.id).any?
```

Comment:
http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-or

forgot about that, it's new to Rails 5